requirements.txt

```diff
@@ -5,13 +5,9 @@
 biopython
 colour==0.1.5
```

Comment:
Doesn't have to be part of this PR, but do you know if we could loosen this one into `>=` as well?

Reply:
We should be able to. `0.1.5` is actually the latest version, so I don't anticipate there being any huge breaking changes in the near future. It's only used in `dash_bio/utils/circos_parser.py` regardless. (Plus, in the `setup.py` install requirements, a version isn't specified so it will default to the latest version anyway.)

Reply:
Let's leave that to a different PR, since you had mentioned needing to change the other dependencies to `>=` as well. (We can do all of them at once!)

Reply:
Actually, I mentioned all the dependencies in the *Dash ecosystem*, so I think we've covered that :)